addons/web/static/src/views/relational_model/record.js

```diff
@@ -243,6 +243,28 @@ export class Record extends DataPoint {
                 parsedValues[fieldName] = staticList;
             } else {
                 parsedValues[fieldName] = this._parseServerValue(field, value);
+                if (field.type === "properties") {
+                    for (const property of parsedValues[fieldName]) {
+                        const fieldPropertyName = `${fieldName}.${property.name}`;
+                        if (property.type === "one2many" || property.type === "many2many") {
+                            const staticList = this._createStaticListDatapoint(
+                                property.value.map((record) => ({
+                                    id: record[0],
+                                    display_name: record[1],
+                                })),
+                                fieldPropertyName
+                            );
+                            parsedValues[fieldPropertyName] = staticList;
```

Comment:
we create a new list each time here, this may be a problem (see above, for the x2many case, we retrieve the list). We can move forward and see later, but this might need more work